start/install/install.md

```diff
@@ -8,29 +8,74 @@ To install to `~/.exo` with no other system changes, run:
 curl -sL https://exo.deref.io/install | bash
 ```
 
-## Package Managers
-
-### MacOS via Homebrew
+## Homebrew (MacOS or Linux)
 
 ```bash
-brew install deref/tap/exo
+brew tap deref/tap
+brew install exo
 ```
 
-See [Deref's Homebrew Tap](https://github.com/deref/homebrew-tap) for the
+See [Deref's Homebrew tap](https://github.com/deref/homebrew-tap) for the
 package source.
 
-### Linux
+## Linux Packages
+
+Linux packages are available for Debian, Alpine, RPM and Homebrew, as well as in a binary format:
+[Latest Release Artifacts](https://github.com/deref/exo/releases/latest).
+
+#### Debian
+
+Download the latest `.deb` file from the release link above and run the
+following command:
+
+```bash
+sudo apt install ./exo_VERSION_linux_ARCH.deb
+```
+
+Note: the file path must be specified either as an absolute path or with a
+leading `./`.
+
+#### Alpine
```

Comment:
I'm not sure if GitBook supports an h4, but we can merge and check.

Reply:
Actually I realise now that these should be h3 anyway.